code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -214,6 +214,14 @@
 		/obj/item/storage/box/visual/magazine = 30,
 		/obj/item/storage/box/visual/grenade = 10,
 		),
+		"Utility" = list(
+		/obj/item/flashlight/combat = -1,
+		/obj/item/binoculars = -1,
+		/obj/item/compass = -1,
+		/obj/item/explosive/plastique = 25,
+		/obj/item/detpack = 15,
```

Comment:
No and no

Reply:
detpacks are fine for me, C4's are bad however, detpacks just explode like grenade or should.